python/GafferImageUI/ImageViewUI.py

```diff
@@ -75,9 +75,13 @@
 	"toolbarLayout:customWidget:StateWidgetBalancingSpacer:section", "Top",
 	"toolbarLayout:customWidget:StateWidgetBalancingSpacer:index", -1,
 
+	"toolbarLayout:customWidget:BottomLeftSpacer:widgetType", "GafferImageUI.ImageViewUI._ExpandingSpacer",
+	"toolbarLayout:customWidget:BottomLeftSpacer:section", "Bottom",
+	"toolbarLayout:customWidget:BottomLeftSpacer:index", 0,
```

Comment:
While the centering looks nicer on it's own, we deliberately removed it before. It causes misalignment with the overlay for the CropWindowTool, and judders around horribly when hopping to a pixel value that formats wider than the usual.

![image](https://user-images.githubusercontent.com/1133871/189314366-82a85a44-4714-4ef7-affd-9005516ffef7.png)

What we want is to align this to the left, as it is in `1.0.x`. I think that's what we'll get if you remove this extra spacer?